kolibri/core/assets/src/core-actions.js

```diff
@@ -83,6 +83,78 @@ function _sessionState(data) {
   return state;
 }
 
+const masteryLoggingMap = {
+  id: 'pk',
+  start_timestamp: 'start_timestamp',
+  completion_timestamp: 'completion_timestamp',
+  end_timestamp: 'end_timestamp',
+  mastery_criterion: 'mastery_criterion',
+  complete: 'complete',
+  responsehistory: 'responsehistory',
+};
```

Comment:
Also weird that that last line got capitalized and an extra space added.

We could generalize this sort of pattern for all API to Store mappings also.